src/views/Farm/components/Stake/DpiFarm.tsx

```diff
@@ -1,129 +1,111 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import React, { useCallback, useMemo, useState } from 'react'
```

Comment:
@0xModene I'm assuming this file is just copy/pasted from the old implementation to restore it?

Reply:
Yes, since I copied ncitron's changes wholly, this was needed.